django/contrib/admin/helpers.py

```diff
@@ -280,16 +280,23 @@ def fields(self):
             if not self.has_change_permission or field_name in self.readonly_fields:
                 yield {
                     'name': field_name,
-                    'label': meta_labels.get(field_name) or label_for_field(field_name, self.opts.model, self.opts),
+                    'label': meta_labels.get(field_name) or label_for_field(
```

Comment:
This can be single-lined.

Reply:
flake8 not pass for long lines. But I remove.

Reply:
Ahh, ok I missed that. Ignore my comment.